backend/pkg/config/config.go

```diff
@@ -207,22 +199,12 @@ func defaultPluginDir() string {
 	if err != nil {
 		logger.Log(logger.LevelError, nil, err, "creating plugins directory")
 
-		return pluginDirDefault
+		return "./.plugins"
```

Comment:
Actually I thinking maybe we just never return this `.plugins` if we are never using it, and we remove it from the comments above as well.

Reply:
made the change, let me know if there's something we are missing here